src/scripts/backfill-usd-price.ts

```diff
@@ -51,6 +51,7 @@ const backfillUsdPrice = async () => {
       await idb.none(pgp.helpers.concat([queries]));
       await wait(6000);
     } catch (err) {
+      await wait(60000);
```

Comment:
Why wait in catch?

Reply:
Also, why are we using `wait`?

Reply:
@shubhampatwa Can we achieve this using bull queue config?

Reply:
this is for just testing, removing not of any use for now